apps/dashboard/src/hooks/use-invocation-queue.ts

```diff
@@ -58,6 +57,18 @@ export function useInvocationQueue<T extends CallbackFunction = CallbackFunction
     [debounceInMs, processQueue]
   );
 
+  const safelyRunInvocation = useCallback(async (invocation: T | undefined) => {
+    if (!invocation) return;
+
+    try {
+      await invocation();
+    } catch (error) {
+      // If the invocation fails, we want to log the error and continue with the next invocation
+      console.error('Error processing queue item:', error);
```

Comment:
I'm not sure if the `console.error` brings any value here.

Reply:
I agree, we can remove it.